### PR TITLE
Suggested changes

### PR DESCRIPTION
**Pull request by T-Nicholls**

### CODE REVIEW

File: ioc/ticker.db [hunk 1]
```diff
@@ -1,15 +0,0 @@
-record(longin, "SR-CS-VEFB-01:TIMER")
```
Comment (aawdls) [inline ioc/ticker.db]: Will we still need these two files (externally) if we want to run with the TFB / VEFB?

File: virtac/README.rst [hunk 2]
```diff
@@ -17,10 +17,10 @@ Pytac, and ATIP.
 
 Inside the top-level atip directory::
 
-    $ ./start-ioc
+    $ ./start-virtac
 
 
-After a minute or two, you should be presented with something like this::
+After a minute or so, you should be presented with something like this::
```
Comment (aawdls) [inline virtac/README.rst]: :laughing:

**Comment by aawdls** [comment]
Will suggested we merge this and use that as a starting point for testing and any subsequent changes.

File: ioc/ticker.db (hunk 1 above)
Comment (T-Nicholls) [reply ioc/ticker.db]: I don't think so. If I remember correctly even with these two "tickers" running externally it didn't work at all for one and the other was borderline and raised an error eventually. So I think we deemed it easier to just comment out the time checking lines in the feedback code instead.

Comment (aawdls) [reply ioc/ticker.db]: Quite right

**Pull request merged**